speechbrain/augment/augmenter.py

```diff
@@ -75,7 +77,7 @@ def __init__(
         max_augmentations=None,
         shuffle_augmentations=False,
         repeat_augment=1,
-        **augmentations,
+        augmentations=list(),
```

Comment:
we should add an assert on the size of `augmentations` (e.g. check that the provided list is not empty)

Reply:
If the list is empty, no augmentation should be performed. Maybe I can raise a warning when that happen.